.gitattributes

```diff
@@ -5,3 +5,6 @@
 # force all text files on the working dir to have LF line endings
 # https://git-scm.com/docs/gitattributes#_eol
 * text eol=lf
+
+*.bat text eol=crlf
+*.jar binary
```

Comment:
Not sure what this does...?

Reply:
It tells git to not touch the line endings.

`.bat` files are meant for Windows, so they should end in CRLF, currently they're LF, because of the top wildcard.

`.jar` files are `binary` file, meaning they are not `text`, line endings (`eol`) are meaningless for binary files. When I committed the new wrapper, it said the next time I do a git operation it'll replace line endings, which means the `.jar` contents would change and the `ZIP` decoder could/would fail.

These two are quite standard and well known in Gradle repos:
https://www.google.com/search?q=*.bat+text+eol%3Dcrlf+*.jar+binary